.vscode/launch.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Checking in my vscode configuration, as you guys are going to want to update yours. It remains in the .gitignore file though, so later changes will not be pulled in. This will become the "base" launch config for vscode.